demystifying_command_line/demystifying_command_line.md

```diff
@@ -0,0 +1,234 @@
+<!--
+module_id: demystifying_command_line
+author:   Joy Payton
+email:    paytonk@chop.edu
+version:  1.0.0
+current_version_description: Initial version
+module_type: standard
+docs_version: 4.0.0
+language: en
+narrator: US English Female
+mode: Textbook
+title: Demystifying the Command Line Interface 
+comment:  Understand what the command line interface is and why it's useful!
+long_description: The command line interface (CLI) of your computer allows you to type commands to do tasks. This module explains what the command line interface is, how to get to it, and how to learn more about using it.  It's appropriate for brand new beginners.
+estimated_time_in_minutes: 30
```

Comment:
```suggestion
estimated_time_in_minutes: 15
```
feels more accurate to me given current cuts, but feel free to push back

Reply:
I agree!